app/javascript/styles/containers.scss

```diff
@@ -9,11 +9,29 @@
   }
 }
 
+.mastodon-column {
+  display: flex;
+  flex: 1 1 auto;
+  padding-bottom: 10px;
+  padding-top: 10px;
+  position: relative;
+
+  .column {
+    overflow-y: auto;
+    padding-bottom: 0;
+    padding-top: 0;
+  }
+}
+
 .mastodon-column-container {
   display: flex;
   flex: 1 1 auto;
 }
 
+.mastodon-column-container--hidden {
+  display: none;
```

Comment:
(reproducible on `Mozilla/5.0 (X11; Linux x86_64; rv:53.0) Gecko/20100101 Firefox/53.0`, with the single-column layout)